sw/host/opentitanlib/rustfmt.toml

```diff
@@ -0,0 +1,5 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+max_width = 80
```

Comment:
I would like to push back gently on this. Do we need to deviate from the defaults?

Reply:
For reference the default is 100. The signer is current set to 80.

Reply:
To me, 80 is established as the default in the signer and is the default in the manticore codebase.

@alphan do you have any opinions on preferred line length for rust code?

Reply:
This is a discussion that should be given slightly more visibility and end up in a at least one-liner style guide for Rust code. (Actually, established practice for such a topic would be an RFC for ultimately the TC to vote on. I'd like to see at least an agreement from all stakeholders.)

Could you do that please, Chris?

Reply:
I don't think this is worth spending the time on.  I'm fine to leave this at the rustfmt default.

80 columns is _my_ default, is Google's C/C++ default and is the PEP8 default (well, 79, but who's counting), so that's why I'd chosen it.

Reply:
@imphil I think an RFC or getting the TC involved is overkill for something like this. In my opinion that route should be reserved for changes that commit us to a course of action that requires a significant commitment from contributors or is difficult to undo. Changes to rustfmt's config do result in churn but are otherwise fairly benign and don't really tie us into anything long term.

Reply:
Let me expand a bit on my previous comment: I'm not saying this must be an RFC. But if we want a project-wide coding style (and enforce it), we need to ensure that we have at least a majority of stakeholders in favor of it. 

Finding such a majority requires that people actually know that there's a decision to be made, and that they feel invited to voice their opinion. Doing so in a quick poll in e.g. the software meeting is fine IMO, as is a separate PR along the lines of "Document Rust style guide" and tagging a relevant contributors, as is an RFC. 

On the other hand, making such a decision in the line comments in a PR that's called "[opentitantool] Apply formatting" doesn't meet the visibility criteria, IMO.

Since we're on the topic: Please don't see the TC as enemy or useless process. It is actually rather lightweight and in most cases, the TC only ensures that all relevant project partners had a chance to weigh in. We can then all safely build on top of that decision without having to go the other way the week later. An RFC doesn't need to be a 100 page document; a single line can do. If you have experienced things to be different let the TC (or me personally) know and we'll make sure to look into that.